src/sys/windows.rs

```diff
@@ -42,6 +42,11 @@ const WSA_FLAG_OVERLAPPED: DWORD = 0x01;
 
 pub use winapi::ctypes::c_int;
 
+/// Fake MSG_TRUNC flag for the [`RecvFlags`] struct.
```

Comment:
Could you add a comment that this is used if `WSAEMSGSIZE` is returned?

Reply:
Added :)